platform/chronos/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG RUST_VERSION=1.80.1
+ARG APP_NAME=discord
+
+FROM lukemathwalker/cargo-chef:latest-rust-${RUST_VERSION} AS chef
+ARG APP_NAME
+WORKDIR /app
+
+FROM chef AS planner
+COPY . .
+
+RUN cargo chef prepare  --recipe-path recipe.json
+
+FROM chef AS builder
+COPY --from=planner /app/recipe.json recipe.json
+
+RUN cargo chef cook --release --recipe-path recipe.json
+COPY . .
+
+ENV SQLX_OFFLINE=true
+ENV APP_NAME=${APP_NAME}
+
+RUN cargo build --release --bin chronos
+
+FROM debian:bookworm-slim AS final
+WORKDIR /app
+RUN apt-get update -y \
+    && apt-get install -y --no-install-recommends openssl ca-certificates \
+    # Clean up
```

Comment:
TODO or remove.

Reply:
yea, this is part of the chef builder for building docker images with rust, found this in the book Zero to Production in Rust, and got a lot of recommendations to use this dockerfile after running into a lot of pain

Reply:
sorry, wrong comment, removing the comment